src/components/MenusAndToolbars/Sheets/Sheets.stories.mdx

```diff
@@ -1,62 +1,136 @@
 import { Meta, Canvas, Story } from "@storybook/addon-docs/blocks";
 import PbSheets from "./Sheets.vue";
+import { colors } from '@pb/utils/constants.js';
 
 <Meta
   title="Components/Data Visualization/PbSheets"
   component={PbSheets}
   argTypes={{
-    enabledToggleButton: { type: "boolean", defaultValue: true },
     sheets: {
-      type: "object",
-      defaultValue: {
-        Bloquear: {
-          icon: "fas fa-ban",
-          title: "Bloquear",
-          disable: true,
+      type: 'array',
+      defaultValue: [
+        {
+          label: 'Compartilhar por e-mail',
+          icon: 'fas fa-envelope',
+          value: 'email',
         },
-        Confirmar: {
-          icon: "fas fa-check",
-          title: "Confirmar",
-          disable: false,
+        {
+          label: 'Enviar por whatsapp',
+          icon: 'fab fa-whatsapp',
+          value: 'whatsapp',
         },
-        Configurar: {
-          icon: "fas fa-cog",
-          title: "Configurar",
-          disable: false,
+        {
+          label: 'Compartilhar link',
+          value: 'link',
         },
-      },
+      ],
+    },
+    buttonIcon: {
+      type: 'string',
+      defaultValue: 'fas fa-ellipsis-v',
     },
-    optionSelected: {
-      action: "optionSelected",
+    buttonColor: {
+      control: { type: 'select', options: [...colors] },
+      defaultValue: 'primary',
+    },
+    buttonStyle: {
+      control: {
+        type: 'select',
+        options: ['regular', 'background', 'outline'],
+      },
+      defaultValue: 'regular',
     },
   }}
 />
 
 export const Template = (args, { argTypes }) => ({
   props: Object.keys(argTypes),
+  methods: {
+    onSelectSheet(value) {
+      console.log('selected sheet', value);
```

Comment:
debug?

Reply:
no, this is to prove that the event works.